### ISSUE TITLE
Clash of '-d' flag in cli commands

### ISSUE THREAD

**Issue by beatrizsavinhas** [Clinical-Genomics/cg#3234]
## Description

The command `cg deliver ticket -t <ticket_id> -d <delivery_type>` fails with error:

```Shell
Try 'cg deliver ticket --help' for help.

Error: Missing option '-d' / '--delivery-type'. Choose from:
	balsamic,
	balsamic-qc,
	balsamic-umi,
	mip-dna,
	mip-rna,
	microsalt,
	fastq,
	mutant,
	rnafusion,
	taxprofiler,
	tomte
```

The error seems to happen because the option for `--dry-run` also allows for the abbreviated flag `-d`, which causes a clash.

## Suggested solution

- Remove the abbreviation '-d' from the dry run option '-d'
- Iden for downsample

## This can be closed when

There is no clash for the '-d' flag in cli commands.

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by ivadym** [comment]
Muttacs `process-solved` is also affected (`"-d", "--days-ago"`).

Could we use `-dt` for the `delivery_type` and `-da` for `--days-ago`? @beatrizsavinhas 

Deliver ticket it's not part of the automation, and `cg upload process-solved` uses [--days-ago](https://github.com/search?q=repo%3AClinical-Genomics%2Fservers+process-solved&type=code)

**Comment by ivadym** [comment]
Would you like to have a look @eliottBo?

**Comment by beatrizsavinhas** [comment]
> Muttacs `process-solved` is also affected (`"-d", "--days-ago"`).
> 
> Could we use `-dt` for the `delivery_type` and `-da` for `--days-ago`? @beatrizsavinhas
> 
> Deliver ticket it's not part of the automation, and `cg upload process-solved` uses [--days-ago](https://github.com/search?q=repo%3AClinical-Genomics%2Fservers+process-solved&type=code)

I guess we could, but I think there are other `-d` flags too in other commands like `demultiplexing`, for example.

**Comment by ivadym** [comment]
I think I went through all of them, and for the rest of the options with `-d` (like `--down-sampled`), we don't have a dry-run, so they should not be affected

**Comment by eliottBo** [comment]
Yes I would like to take a look. Is it better to change the other flags that use now -d as a flag than to change the dry-run one?

**Comment by henrikstranneheim** [comment]
> Yes I would like to take a look. Is it better to change the other flags that use now -d as a flag than to change the dry-run one?

Yes I think so. I prefer non standard Unix flag (i.e, not `-v`, `-h`) to have verbose names so they are mnemonic. For example: `--delivery-type`. These do not need a short flag because it can cause clashes and are hard to remember.
